### ISSUE TITLE
Can I configure a different port?

### ISSUE THREAD

**Issue by dgracehmh** [palantir/bulldozer#170]
Hi, 

I want to deploy bulldozer as a docker container on our cloud infrastructure (Apache Mesos/Aurora). Aurora dynamically assigns a port to the running container on deploy. Is there any way to run the server with a port other than 8080 (defined in the bulldozer config file)? It would solve my issue if I could pass that port in as an environment variable.
Thanks

**Comment by asvoboda** [comment]
Hey @dgracehmh,

I'm not familiar with mesos configuration, so ymmv. However, we only specify the port for bulldozer to bind to within the container. You can map the exposed port to any external port you'd like on the host.

This functionality is similar to the way `docker run -P` works, by assigning a random ephemeral port on the host to bind to the exposed port by docker.

**Comment by dgracehmh** [comment]
hi @asvoboda . Apache aurora works but assigning a dynamic port to a job, that port then needs to be passed to the service which can then be exposed on 8080. Something like 
```
nginx = Process(
  name = 'nginx',
  cmdline = './run_nginx.sh -port {{thermos.ports[http]}}'
)
```
Or it can also be exposed to the app running in docker through environment variables.
See - http://aurora.apache.org/documentation/latest/features/services/

**Comment by bluekeyes** [comment]
Sorry for the long delay here, but as of #220 (released in 1.12.0), you can now set the port via the `BULLDOZER_PORT` environment variable if you can't or don't want to provide a different configuration file.
